apps/metadata-editor-e2e/src/e2e/edit.cy.ts

```diff
@@ -57,4 +57,49 @@ describe('editor form', () => {
     })
     cy.get('md-editor-publish-button').click()
   })
+
+  describe('Access and contact', () => {
+    describe('Open data switch', () => {
+      describe('When the open data switch is checked', () => {
+        beforeEach(() => {
+          cy.visit('/edit/accroche_velos')
+          cy.get('md-editor-page-selector').find('gn-ui-button').last().click()
+        })
+        it('should not display the licence section', () => {
+          cy.get('gn-ui-form-field-license').should('not.exist')
+        })
+        it('should display the license section when toggled', () => {
+          cy.get('gn-ui-check-toggle').find('span').first().click()
```

Comment:
Fair enough, will add a data-cy selector ;)

I think I did try to click on the toggle directly, because it would be strange to complicate the test directly without doing so. With the selector it might work directly